activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -314,6 +314,12 @@ def supports_pgcrypto_uuid?
         postgresql_version >= 90400
       end
 
+      # PostgreSQL 9.4 introduces ALTER TABLE ... ALTER CONSTRAINT but it has a bug and fixed in 9.4.2
+      # https://www.postgresql.org/docs/9.4/static/release-9-4-2.html
```

Comment:
Moved and force pushed!

BTW, I followed the same way as `supports_ranges?` so it may not be intended.
I created another PR https://github.com/rails/rails/pull/28543.
Please close the PR if the comment is intended :sweat_smile: